src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -80,7 +55,8 @@ private static String getTenant(String nsTenant) {
     context.runOnContext(v -> {
       PostgresClient pgClient = PostgresClient.getInstance(vertx, tenant);
       pgClient.setIdField("_id");
-      String query = String.format("(status == \"%s\" AND requestExpirationDate < \"%s\") OR (status == \"%s\" AND holdShelfExpirationDate < \"%s\")",
+      String query = String.format(
+        String.format("%s OR %s", QUERY_TO_LOOK_FOR_EXPIRED_UNFILLED, QUERY_TO_LOOK_FOR_EXPIRED_AWAITING_PICKUP),
```

Comment:
Thank you, that makes the distinction between the two more explicit

Reply:
thank you!